Bottom: d46669691e365aa3a324c3eeb50bdf3697e8da7a
Top:    638ddfe573607e47b918fa7b027ca1edc195fb9a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-24 00:15:26 +0000

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/change.c b/src/change.c
index 3791ccc..a6ee486 100644
--- a/src/change.c
+++ b/src/change.c
@@ -457,7 +457,6 @@ ChangeVia2ndSize (PinTypePtr Via)
     {
       AddObjectTo2ndSizeUndoList (VIA_TYPE, Via, Via, Via);
       EraseVia (Via);
-      RestoreToPolygon (PCB->Data, VIA_TYPE, Via, Via);
       RestoreToPours (PCB->Data, VIA_TYPE, Via, Via);
       Via->DrillingHole = value;
       if (TEST_FLAG (HOLEFLAG, Via))
